javagen/src/main/java/com/azure/autorest/util/TemplateUtil.java

```diff
@@ -81,9 +79,12 @@ public class TemplateUtil {
      * @return the JSON string
      */
     public static String prettyPrintToJson(Object jsonObject) {
```

Comment:
I guess this is no longer "pretty print".

As long as SDK CI won't report anything, I am fine with the formatting of json file.

Reply:
I haven't run across issues with this being removed. Unless this is something I need to regenerate an actual Azure SDK with to verify.